commands/upgrade/breadcrumbs.py

```diff
@@ -59,6 +110,8 @@ def save(self):
                 crumbs.truncate()
                 json.dump(doc, crumbs, indent=2, sort_keys=True)
                 crumbs.write('\n')
+                if os.environ.get('LEAPP_NO_RHSM_FACTS', '0') == '0':
+                    self._save_rhsm_facts(doc['activities'])
```

Comment:
This is also affected in situation when `LEAPP_NO_RHSM_FACTS=`, it's better to check the valus is not equal `1` so the function is not called iff `LEAPP_NO_RHSM_FACTS=1` 
```suggestion
                if os.environ.get('LEAPP_NO_RHSM_FACTS', '0') != '1':
                    self._save_rhsm_facts(doc['activities'])
```

Reply:
Suggestion applied :)